src/public/server.js

```diff
@@ -322,15 +322,8 @@ Dungeon.prototype = {
         console.log("Error can't move " + direction);
     }
     
-    // check forbidden movement
-    if( 
-      (requestedX < 0) ||
-      (requestedY < 0) ||
-      (requestedX >= this.area.columns) ||
-      (requestedY >= this.area.rows)
-    ) {
-      return;
-    }
+    requestedX = requestedX % this.area.columns + 1 ? requestedX % this.area.columns : this.area.columns - 1;
```

Comment:
Hmmm ... It seems to me that `requestedX % this.area.columns + 1` is only  `false` when `requestedX == this.area.columns - 1` which implies that `requestedX % this.area.columns = this.area.columns - 1` then your expression output is always `requestedX % this.area.columns`

Reply:
Maybe you could had a comment there to explain that this transformation is a way to ensure 'area continuity'

Reply:
Ok, i tested and i understood my mistake. Don't know how to make it simpler for now.